tests/integration/components/nypr-accounts/membership-card-test.js

```diff
@@ -31,16 +31,30 @@ module('Integration | Component | nypr accounts/membership card', function(hooks
     assert.ok(find('.pledge-help-link'), 'has help link');
   });
 
-  test('it displays the redirect message when the pledge manager is disabled', async function(assert) {
-    await render(hbs`{{nypr-accounts/membership-card pledgeManagerEnabled=false}}`);
+  test('it displays the custom message and link', async function(assert) {
+    let message = "Test 123"
```

Comment:
can these variables be `const`? If so, changing them can help a new developer understand that those are not modified (I tend to assume that `let` is intentionally used to indicate that the variable may change by logic).

Reply:
No problem.